core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-history/dot-container-history.component.spec.ts

```diff
@@ -36,12 +36,16 @@ describe('ContainerHistoryComponent', () => {
 
     describe('history', () => {
         beforeEach(() => {
+            component.containerId = '123';
+            component.ngOnInit();
```

Comment:
This is a big no no in testing, `fixture.detectChanges()` does the init.

Reply:
updated